server/src/test/java/org/elasticsearch/search/suggest/SuggestTests.java

```diff
@@ -214,4 +223,39 @@ public void testMergingSuggestionOptions() {
         assertTrue(option1.getScore() > 0.7f);
         assertTrue(option1.collateMatch());
     }
+
+    public void testSerialization() throws IOException {
+        final Version bwcVersion = VersionUtils.randomVersionBetween(random(),
+            Version.CURRENT.minimumCompatibilityVersion(), Version.CURRENT);
```

Comment:
My thinking was that writing it like this, it will still make sense regardless of what the current version is. For example, there are plenty of tests like this hanging around that explicitly test versions that are now outside of the compatibility range, and I was thinking this is a way of avoiding that